pkg/proto/nbcontract/v1/gpuconfig.proto

```diff
@@ -4,23 +4,12 @@ package nbcontract.v1;
 import "pkg/proto/nbcontract/v1/featurestate.proto";
 
 message GPUConfig {
-  // NvidiaState represents the Nvidia GPU state
-  optional FeatureState nvidia_state = 1;
-
-  // FeatureState is_mig_node = 2; removed since it only depends on gpu_instance_profile
-
   // ConfigGpuDriver specifies whether bootstrap process should install and configure the GPU driver when necessary. Configuration includes appropriate set up of components like the fabric manager where applicable.
-  optional FeatureState config_gpu_driver = 2;
+  FeatureState config_gpu_driver = 1;
```

Comment:
Do we need FeatureState for config_gpu_driver and gpu_device_plugin? Or simply bool is enough?
FeatureState was used here before the FeatureState vs bool conversation happened. We can make a decision here.

Reply:
Yes good catch, I think these can be bools.